sklearn/utils/metaestimators.py

```diff
@@ -10,11 +10,47 @@
 __all__ = ['if_delegate_has_method']
 
 
+def hasattr_nested(obj, attr):
+    """Check if obj has a (possibly nested) attribute
```

Comment:
Okay. The docstring is too verbose. This function will probably not be used anywhere else. Can you just describe what it does in the docstring without examples?